ada_2_fhir-r4/zibs2020/payload/0.0/zib-AddressInformation.xsl

```diff
@@ -0,0 +1,170 @@
+                                        <display value="Tijdelijk adres"/>
+                                    </coding>
+                                </valueCodeableConcept>
+                            </extension>
+                            <use value="temp"/>
+                        </xsl:when>
+                        <!-- Werkadres -->
+                        <xsl:when test="@code = 'WP'">
+                            <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-zib-AddressInformation-AddressType">
+                                <valueCodeableConcept>
+                                    <coding>
+                                        <system value="{normalize-space(adres_soort/@codeSystem)}"/>
+                                        <code value="WP"/>
+                                        <display value="Werkadres"/>
+                                    </coding>
+                                </valueCodeableConcept>
+                            </extension>
+                            <use value="work"/>
+                        </xsl:when>
+                        <!-- Vakantie adres -->
+                        <xsl:when test="@code = 'HV'">
+                            <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-zib-AddressInformation-AddressType">
+                                <valueCodeableConcept>
+                                    <coding>
+                                        <system value="{normalize-space(adres_soort/@codeSystem)}"/>
+                                        <code value="HV"/>
+                                        <display value="Vakantie adres"/>
+                                    </coding>
+                                </valueCodeableConcept>
+                            </extension>
+                        </xsl:when>
+                    </xsl:choose>
+                </xsl:for-each>
+                <xsl:if test="$lineItems">
+                    <line>
+                        <xsl:copy-of select="$lineItems"/>
+                    </line>
+                </xsl:if>
+                <xsl:if test="woonplaats/@value">
+                    <city value="{normalize-space(woonplaats/@value)}"/>
+                </xsl:if>
+                <xsl:if test="gemeente/@value">
+                    <district value="{normalize-space(gemeente/@value)}"/>
+                </xsl:if>
+                <xsl:if test="postcode/@value">
+                    <postalCode value="{normalize-space(postcode/@value)}"/>
+                </xsl:if>
+                <!--Dit kan volgens mij beter via een template (TODO)-->
+                <xsl:if test="land/@code">
+                    <country value="{normalize-space(land/@displayName)}">
+                        <extension
+                            url="http://nictiz.nl/fhir/StructureDefinition/ext-CodeSpecification">
+                            <valueCodeableConcept>
```

Comment:
Ook hier kan je code-to-CodeableConcept gebruiken.

Reply:
Used the template (code-to-CodeableConcept)